content/TruenasApps/StableApps/EmbyApp.md

```diff
@@ -0,0 +1,189 @@
+---
+title: "Emby"
+description: "Provides instructions to configure TrueNAS and install Emby media server app for movie, TV shows, books, photos, music, and other media files."
+weight:
+aliases:
+tags:
+- apps
+- media
+keywords:
+- nas data storage
+- software storage solutions
+---
+
+Emby manages personal media libraries of home videos, movies, TV shows, photos, books, and music files.
+Emby has two parts with at least one client and server.
+The client or apps play back media files stored in the Emby server, and the server stores the media library.
+TrueNAS Emby app provides the server function.
+The Emby service uses a Web UI to manage user profiles that are allowed to access your media files and allows you to manage the media library of files.
+Emby naming conventions for media files are important. Pay close attention to the instructions Emby provides to correctly configure the TrueNAS Emby app storage volumes, and when adding any labels to apply to media files.
+
+{{< include file="/static/includes/AppsUnversioned.md" >}}
+
+## Before You Begin
+Before you install the Emby app:
+
+* Read the [Emby Support documentation](https://emby.media/support/articles/Home.html).
+  Emby instructions in the **Media Preparation** and [**Library Setup**](https://emby.media/support/articles/Library-Setup.html) sections, influence how you set up storage volume types and labels used in the TrueNAS **Install Emby** app wizard.
+  Organize media files into folders by type, movies, photos, music, etc.
+  Emby instructions say to include the release date of movies in the folder name as follows, *movieName (year)*.
+
+{{< include file="/static/includes/apps/BeforeYouBeginStableApps.md" >}}
+{{< include file="/static/includes/apps/BeforeYouBeginRunAsUser.md" >}}
+
+<div style="margin-left: 33px">{{< trueimage src="/images/SCALE/Apps/EmbyDetailsScreen.png" alt="Nextcloud App Details Screen" id="Nextcloud App Details Screen" >}}</div>
+
+{{< include file="/static/includes/apps/BeforeYouBeginAddNewAppUser.md" >}}
+<div style="margin-left: 33px">The Emby run-as user ID is **0**. The app also includes the default **apps** user ID **568**.
```

Comment:
These asterisks aren't rendering as bolded in my build, just plain asterisks.

Reply:
replaced with html tags.